protocol-definitions/Map.yaml

```diff
@@ -2942,3 +2942,27 @@ methods:
           since: 2.0
           doc: |
             old value of the entry
+  - id: 72
+    name: replaceAll
+    since: 2.0
+    doc: |
+      Replaces each entry's value with the result of invoking the given
+      function on that entry until all entries have been processed or the
+      function throws an exception.
+    request:
+      retryable: true
+      partitionIdentifier: -1
```

Comment:
Can you put `partitionId` as `partitionIdentifier` . 
Also on the `doc` , we need to mention that this will replace all entries in the targeted partition only.